search/serializers.py

```diff
@@ -86,6 +87,48 @@ def postprocess_fields(self, discussions_obj, serialized_data):
         return {"author_channel_membership": sorted(get_channels(discussions_obj.user))}
 
 
+class ESCourseSerializer(ESSerializer):
+    """
+    Elasticsearch serializer class for courses
+    """
+
+    object_type = COURSE_TYPE
+    use_keys = [
+        "course_id",
+        "short_description",
+        "full_description",
+        "platform",
+        "language",
+        "semester",
+        "year",
+        "level",
+        "start_date",
+        "end_date",
+        "enrollment_start",
+        "enrollment_end",
+    ]
+    rename_keys = {"title": "course_title", "image_src": "course_image"}
```

Comment:
Is there a rationale behind renaming these particular fields? It seems like `short_description` could arguably be prefixed by "course_" too, so I'm trying to understand if there's a functional reason to this.

Reply:
These two just seemed a bit too short & generic to me, I thought it might be good to more specifically differentiate them from similar fields like `post_title` for example.  I can keep the original field names if you prefer.